lib/mongoid/extensions/typed_array_class_factory.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Mongoid
+  module Extensions
+
+    # A factory for manufacturing Mongoid::<Type>Array classes.
+    module TypedArrayClassFactory
```

Comment:
This module should be `@api private` also methinks.

Reply:
done